spec/ruby/core/process/times_spec.rb

```diff
@@ -17,20 +17,17 @@
     end
   end
 
-  platform_is_not :windows do
+  guard -> do
+    Process.clock_gettime(:GETRUSAGE_BASED_CLOCK_PROCESS_CPUTIME_ID)
+  rescue
+    skip "getrusage is not supported on this environment"
```

Comment:
Do we need to skip it inside `guard`? I thought you're supposed to return false.